fluentd/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM unit9/base:latest
+
+COPY "fluentd.run" "/etc/service/fluentd/run"
```

Comment:
Can you please put the `COPY` statements for things that do not affect the `apt-get install`, AFTER the `apt-get install` stuff? This way tweaking a `.run` file doesn't invalidate the cached install.